src/ports/postgres/modules/knn/test/knn.sql_in

```diff
@@ -0,0 +1,41 @@
+m4_include(`SQLCommon.m4')
+/* -----------------------------------------------------------------------------
+ * Test knn.
+ *
+ * FIXME: Verify results
```

Comment:
I got the license, thanks!
For assertions, I was trying doing that yesterday. It was not working.
For example, I tried doing 
SELECT assert(3 = 3, 'Wrong output in pivoting');
in postgres prompt
and it says ''HINT:  No function matches the given name and argument types. You might need to add explicit type casts."
Can you tell what is happening here. I am using postgres 9.4.